models/app.model.js

```diff
@@ -95,4 +95,24 @@ async function updateArticleModel (inc_votes, chosenId) {
 
 }
 
-module.exports = {getAllTopics, getArticleById, getAllArticlesModel, getAllCommentsForArticleModel, postCommentToArticleModel, updateArticleModel}
\ No newline at end of file
+async function deleteCommentModel (chosenId) {
+    const queryResult = await db.query(`DELETE FROM comments WHERE comment_id = $1 RETURNING *`, [chosenId]);
+
+
+    if (queryResult.rows.length < 1) {
+        let result = await checkExists("comments", "comment_id", chosenId)
```

Comment:
We don't need to check if the comment exists first, we can look at the response from the delete query to determine if it has successfully deleted a comment or not.